<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>genParamap</class>
 <widget class="QWidget" name="genParamap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Parametric Map</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(42, 42, 42);
}</string>
  </property>
  <widget class="QLabel" name="saveRoiLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>-30</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 29px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Generate Parametric Map:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="roiFolderPathLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>130</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(255, 255, 255, 0);
	color: white;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string>Folder to save Paramap in:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newFolderPathInput">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>180</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(249, 249, 249);
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseFolderButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Choose Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearFolderButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Clear Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateParamapButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataSavedSuccessfullyLabel">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>230</y>
     <width>551</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(23, 255, 0);
	font-size: 30px;
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Parametric map generated successfully!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="newFileNameLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgba(255, 255, 255, 0);
	color: white;
	font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string>Type name of new file:
(no spaces,  must end in &quot;.nii.gz&quot;)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newFileNameInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(249, 249, 249);
	color: black;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fileNameErrorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(255, 0, 23);
	font-size: 20px;
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>ERROR: Filename must not contain a space
and must end in &quot;.nii.gz&quot;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fileNameWarningLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(255, 0, 23);
	font-size: 20px;
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>WARNING: If file of specified folder and
filename already exists,  that file will be
deleted and overwritten</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="axOverlapLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Axial Overlap (%)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imageWidthVal">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="axWinSizeLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Axial Window Size (mm)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="axWinSizeVal">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: white;
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="latWinSizeVal">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: white;
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="latWinSizeLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Lateral Window Size (mm)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="latOverlapLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>Lateral Overlap (%)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imageWidthLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>CE Image Width (mm)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="axOverlapVal">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color: white;
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="latOverlapVal">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>430</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color: white;
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageDepthLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>CE Image Depth (mm)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imageDepthVal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="latOverlapLabel_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>691</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>NOTE: This may take several minutes to run depending on data size and Overlap</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
